digitaltwins/samples/device/javascript/pnpSampleDevice.js

```diff
@@ -0,0 +1,202 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+'use strict';
+
+var Protocol = require('azure-iot-device-mqtt').Mqtt;
+var Client = require('azure-iot-device').Client;
+var Message = require('azure-iot-device').Message;
+
+const packageJson = require('./package.json');
+
+// String containing Hostname, Device Id & Device Key in the following formats:
+//  'HostName=<iothub_host_name>;DeviceId=<device_id>;SharedAccessKey=<device_key>'
+const deviceConnectionString = process.env.DEVICE_CONNECTION_STRING;
+
+const modelId = 'dtmi:com:example:SampleDevice;1';
+const messageSubjectProperty = '$.sub';
+const sensorComponentName = 'sensor';
+const deviceInfoComponentName = 'deviceInformation';
+const sdkInfoComponentName = 'sdkInformation';
+const componentPrefix = '$iotin:';
+const commandComponentCommandNameSeparator = '*';
+
+const commandNameBlink = componentPrefix + sensorComponentName + commandComponentCommandNameSeparator + 'blink';
+const commandNameTurnOn = componentPrefix + sensorComponentName + commandComponentCommandNameSeparator + 'turnOn';
+const commandNameTurnOff = componentPrefix + sensorComponentName + commandComponentCommandNameSeparator + 'turnOff';
+
+const propertyUpdateHandler = (deviceTwin, componentName, propertyName, reportedValue, desiredValue, version) => {
+  console.log('Received an update for component: ' + componentName + ' and property: ' + propertyName + ' with value: ' + JSON.stringify(desiredValue));
+  const patch = helperCreateReportedPropertiesPatch(
+    { [propertyName]: desiredValue}, 
+      componentName,
+      {
+        code: 200,
+        description: 'Successfully executed patch for ' + propertyName,
+        version: version
+      }
+    )
+  updateComponentReportedProperties(deviceTwin, patch, componentName);
+  console.log('updated the property');
+};
+
+
+const commandHandler = async (request, response) => {
+  switch (request.methodName) {
+  case commandNameBlink: {
```

Comment:
Add a console.log here just so users can see that blink happened.

Reply:
the send method actually calls a helper which logs out the command.